src/pushprocessor.ts

```diff
@@ -90,6 +90,73 @@ const DEFAULT_OVERRIDE_RULES: IPushRule[] = [
         ],
         actions: [],
     },
+    {
+        // Matrix v1.7
+        rule_id: ".m.rule.is_user_mention",
+        default: true,
+        enabled: true,
+        conditions: [
+            {
+                kind: ConditionKind.EventPropertyContains,
+                key: "content.m\\.mentions.user_ids",
+                value: "[the user's Matrix ID]",
```

Comment:
erm; is this actually going to work? Don't we need to substitute the user id here?

There is some magic for handling `is_user_mention` in `pushprocessor.ts`. maybe we need to extend it?